### PR TITLE
Fix creation of private gateway with NSX when LSWitch does not exist

### PR DESCRIPTION
**Pull request by miguelaferreira**
The NSX guru now refuses to design the network if the specified LSwitch does not exist.
In addition to that, the HTTP client connection limits were adjusted.

### CODE REVIEW

File: api/src/main/java/com/cloud/exception/InsufficientCapacityException.java [hunk 1]
```diff
@@ -38,6 +38,9 @@ public InsufficientCapacityException(String msg, Class<?> scope, Long id) {
         this.id = id;
     }
 
+    public InsufficientCapacityException(String message, Throwable cause) {
+    }
```
Comment (wilderrodrigues) [inline api/src/main/java/com/cloud/exception/InsufficientCapacityException.java]: Shouldn't we call the super constructor here?

**Comment by wilderrodrigues** [comment]
There is one unit test failing.

https://beta-jenkins.mcc.schubergphilis.com/job/cosmic/job/9998-maven-build/156/cloud.cosmic$cloud-plugin-network-nvp/testReport/com.cloud.network.guru/NiciraNvpGuestNetworkGuruTest/testDesign/

File: api/src/main/java/com/cloud/exception/InsufficientCapacityException.java (hunk 1 above)
Comment (miguelaferreira) [inline api/src/main/java/com/cloud/exception/InsufficientCapacityException.java]: I'll remove that, cause it's not needed

**Comment by miguelaferreira** [comment]
go build

**Comment by miguelaferreira** [comment]
Build fixed, and code is cleaner.

**Pull request merged**